### PR TITLE
find . -type f -exec sed -i s/James/Haldis/g {} \;

### PR DESCRIPTION
**Pull request by Donvittorio**

### CODE REVIEW

**Comment by Donvittorio** [comment]
this fixes #31

File: app/templates/layout.html [hunk 1]
```diff
@@ -72,8 +72,8 @@
     <hr>
     <div class="container">
         <div class="pull-right">Made with ❤ by <a href="http://zeus.ugent.be">Zeus WPI</a></div>
-        <div class="pull-left"><a class="github-button" href="https://github.com/ZeusWPI/james" data-icon="octicon-star" data-style="mega" data-count-href="/ZeusWPI/James/stargazers" data-count-api="/repos/ZeusWPI/James#stargazers_count" data-count-aria-label="# stargazers on GitHub" aria-label="Star ZeusWPI/James on GitHub">Star</a></div>
-        <div class="text-center"><a href="http://github.com/ZeusWPI/James">© {{ ""|year }}</a></div>
+        <div class="pull-left"><a class="github-button" href="https://github.com/ZeusWPI/james" data-icon="octicon-star" data-style="mega" data-count-href="/ZeusWPI/Haldis/stargazers" data-count-api="/repos/ZeusWPI/Haldis#stargazers_count" data-count-aria-label="# stargazers on GitHub" aria-label="Star ZeusWPI/Haldis on GitHub">Star</a></div>
```
Comment (feliciaan) [inline app/templates/layout.html]: Still contains james.

Comment (Donvittorio) [inline app/templates/layout.html]: not anymore

**Comment by Donvittorio** [comment]
2015-04-03-202818_1198x485_scrot
Proof shit works

**Pull request merged**